packages/runtime/container-runtime/src/orderedClientElection.ts

```diff
@@ -364,23 +460,39 @@ export class OrderedClientElection
     }
 
     public incrementElectedClient(sequenceNumber: number): void {
```

Comment:
nit: Not part of this PR but this function should be called `electNextEligibleClient` or something that conveys the same.